docs/components/modeler/element-templates/element-template-with-dependencies.md

```diff
@@ -0,0 +1,35 @@
+---
+id: element-templates-with-dependencies
+title: Element templates with dependencies
+description: "Learn what you need to take into account when handling template dependencies."
+---
+
+When creating element templates, you may want to link to a particular resource like a [form](/components/modeler/forms/camunda-forms-reference.md), or pre-populate a [secret](/components/connectors/use-connectors/index.md#using-secrets) expression. Your template might require a particular [job worker](/components/concepts/job-workers.md) to execute an action. These are all different types of dependencies.
+
+Element templates can depend on:
+
+- Resources:
+  - [Camunda forms](/components/modeler/forms/camunda-forms-reference.md): used in user tasks.
+  - [RPA scripts](/components/rpa/overview.md): Used in service tasks.
+  - [BPMN tasks](/components/modeler/bpmn/bpmn.md): used in call activities.
+  - [DMN decisions](/components/modeler/dmn/dmn.md): used in business rule tasks.
+- [Call activities](/components/modeler/bpmn/call-activities/call-activities.md): this may introduce nested dependencies (For example, a called process may depend on other processes and/or dependencies).
```

Comment:
~~❓ indentation error?~~ My point was actually to adapt the picture to look like the list you had previously 😆. 
🔧 Also, I suppose the actual resource would be called process. So the grouping here doesn't really work.

Reply:
I'll revert the grouping